cmd/cli/main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"log"
 
-	bf "github.com/mainflux/mainflux/bashflux"
+	bf "github.com/mainflux/mainflux/cli"
```

Comment:
Please omit package name if there is no special reason to use `bf` instead of `cli`.

Reply:
Good spot, will change this.